src/allmydata/storage_client.py

```diff
@@ -994,11 +994,19 @@ def slot_testv_and_readv_and_writev(
             tw_vectors,
             r_vector,
     ):
+        # Match the wire protocol, which requires 4-tuples for test vectors.
+        wire_format_tw_vectors = {
+            key: (
+                [(start, length, b"eq", data) for (start, length, data) in value[0]],
+                value[1],
+                value[2],
+            ) for (key, value) in tw_vectors.items()
```

Comment:
So previously callers supplied a 4 tuple to this Python API and the implementation just passed it onto the network.  Now callers must (and have been updated to) supply a 3 tuple that omits the operator (since it must be "eq") and this code translates it to the wire format which is the same as ever for interop between old and new versions of Tahoe.

Just double-checking my understanding since I have some code somewhere else that this will break.

Reply:
That is correct.